### ISSUE TITLE
Second Changes: Close-Knit Flower Sack

### ISSUE THREAD

**Issue by vhellstein** [DenverArtMuseum/biophilia-pub#51]

**Comment by vhellstein** [comment]
- [ ] @1000camels Darrin decided the font was too big. See Poem scale and design example: [](https://emergencemagazine.org/poem/waterline/ ) and h[ttps://emergencemagazine.org/poem/to-the-fig-tree-on-9th-and-christian/](url)

- [ ] @mpopke Subtitle/or direction with audio: “Listen to Poem, Narrated by Cedar Sigo”
- [ ] @1000camels the line spacing is pretty tight when a line wraps. Please add a bit more space.


![Image](https://github.com/DenverArtMuseum/biophilia-pub/assets/119438504/1ca1c2d1-07bb-4c37-91fa-701cd94439bd)

- [ ] @1000camels My apologies about the "block quote" deletion--we do need to keep the spacing. See the indentation in the word doc.

[Close_Knit_Flower_Sack_FINAL.docx](https://github.com/DenverArtMuseum/biophilia-pub/files/14438234/Close_Knit_Flower_Sack_FINAL.docx)

- [ ] @1000camels There is a possibility that we might try to source an image for the background of this page, so the poem would be on top of it. Stay tuned for that...

**Comment by 1000camels** [comment]
I have added a nowrap setting, which helps if someone's font is enlarged. On mobile, I shrunk it down, so it does not wrap, but it is too small. Not sure what to do.

**Comment by 1000camels** [comment]
Just get me the image and I can add it. We can fade it if it conflicts with reading the poem

**Comment by 1000camels** [comment]
I had a look at the Word doc and I am confused. Is the last half of the poem indented by half of the page?

**Comment by mpopke** [comment]
Yes, it is indented pretty significantly. It looks like they used actual spaces instead of tab stops, too. Might be easiest to just copy and paste directly into a div, and use white-space: pre;

**Comment by vhellstein** [comment]
> I have added a nowrap setting, which helps if someone's font is enlarged. On mobile, I shrunk it down, so it does not wrap, but it is too small. Not sure what to do.

@1000camels I'll let Darrin know that it's really either that the text wraps or it's small/you have to scroll to the right on mobile.

**Comment by 1000camels** [comment]
I have added the pre as @mpopke suggested
